clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1214,7 +1214,13 @@ DerivedArgList getLinkerArgs(ArrayRef<OffloadFile> Input,
   // Forward '-Xoffload-linker' options to the appropriate backend.
   for (StringRef Arg : Args.getAllArgValues(OPT_device_linker_args_EQ)) {
     auto [Triple, Value] = Arg.split('=');
-    if (Value.empty())
+    llvm::Triple TT(Triple);
+    // If this isn't a recognized triple then it's an `arg=value` option.
+    if (TT.getArch() <= Triple::ArchType::UnknownArch ||
+        TT.getArch() >= Triple::ArchType::LastArchType)
```

Comment:
should it be `TT.getArch() >= Triple::ArchType::LastArchType` as the last arch type is valid, which is `ve`?

Reply:
Good catch
```suggestion
        TT.getArch() > Triple::ArchType::LastArchType)
```